### ISSUE TITLE
no-unknown-animations is missing single-transition-timing-function "ease"

### ISSUE THREAD

**Issue by Arkkimaagi** [stylelint/stylelint#1240]
I found (what I think is) a bug where the standard `ease` timing function gets flagged as unknown animation name.

It's in the rule `no-unknown-animations` obviously.

AFAIK this should be valid, but causes an error:
```css
@keyframes foobar {
  0% { opacity: 0; }
  100% { opacity: 1; }
}

.foo{
  animation: foobar 1s ease 0.2s 1 both;}
}
```

With these settings:
```YAML
---
rules:
  no-unknown-animations: true
```

Appears at least on stylelint 6.2.2 from command line.
```bash
$ stylelint test.css


test.css
 7:24  ✖  Unknown animation name "ease"   no-unknown-animations
```

I expected to have no warnings, as `ease` is according to [specs](https://developer.mozilla.org/en-US/docs/Web/CSS/animation) referenced in the code below:

[no-unknown-animations/index.js](https://github.com/stylelint/stylelint/blob/master/src/rules/no-unknown-animations/index.js):
```javascript
// cf. https://developer.mozilla.org/en-US/docs/Web/CSS/animation
const animationShorthandKeywords = new Set([
  "infinite", "normal", "reverse", "alternate", "alternate-reverse",
  "none", "initial", "inherit", "unset", "forwards", "backwards", "both", "running", "paused",
  "linear", "ease-in", "ease-out", "ease-in-out", "step-start", "step-end",
])
```

I think `"ease"` should be included in that list. 

As a sidenote, I'm not sure if `"initial", "inherit", "unset"` should be there, they were not mentioned in the specs.

**Comment by evilebottnawi** [comment]
@Arcanemagus Thanks!

**Comment by davidtheclark** [comment]
@Arkkimaagi should be a really easy fix if you're interested in starting a PR.

**Comment by evilebottnawi** [comment]
Done in https://github.com/stylelint/stylelint/pull/1245
